### ISSUE TITLE
Joint axes are incorrect.

### ISSUE THREAD

**Issue by rohit-kumar-j** [yanshil/Fusion2PyBullet#6]
Hi! Thank you for this Addin! 
It saves a lot of time, as before we had to export each mesh file in a separate format(via [fusion2urdf](https://github.com/syuntoku14/fusion2urdf)) in-order for Pybullet to accept the mesh file format. This takes care of it instantaneously. 

Although I ran into a joint axes mismatch (below). I think that you can use the mesh exporter that you have here, and [fusion2urdf's](https://github.com/syuntoku14/fusion2urdf) original code for joint axes and urdf creation and it would be perfect
 
I have also provided the video of how it should actually look like (which was made with [fusion2urdf](https://github.com/syuntoku14/fusion2urdf))

Output: (Please excuse the music. This is my first time using screenrecord application )

Fusion to Pybullet Video:

https://user-images.githubusercontent.com/37873142/107810131-33ffff80-6d92-11eb-9963-c85f4fce3e16.mp4

Fusion to URDF Video:

https://user-images.githubusercontent.com/37873142/107812529-d372c180-6d95-11eb-8b6b-d6acb15d1c80.mp4

My modified hello_bullet.py code, video output:
```
#!/usr/bin/env python3
import pybullet as p
import time
import pybullet_data
physicsClient = p.connect(p.GUI)#or p.DIRECT for non-graphical version
p.setAdditionalSearchPath(pybullet_data.getDataPath()) #optionally
p.setGravity(0,0,0)
# planeId = p.loadURDF("plane.urdf")
cubeStartPos = [0,0,0]
cubeStartOrientation = p.getQuaternionFromEuler([0,0,0])
robotId = p.loadURDF("V14_URDF.urdf",cubeStartPos, cubeStartOrientation, 
                   # useMaximalCoordinates=1, ## New feature in Pybullet
                   useFixedBase=True,
                   flags=p.URDF_USE_INERTIA_FROM_FILE)
val = 0 

for i in range (10000):
    p.stepSimulation()
    time.sleep(1./240.)
    # for j in range(p.getNumJoints(robotId)):
    	# p.setJointMotorControl2(robotId,j,controlMode=p.POSITION_CONTROL,targetPosition=val)
    	# val = val +0.001
    p.setJointMotorControl2(robotId,2,controlMode=p.POSITION_CONTROL,targetPosition=val)
    val = val +0.001
    p.setJointMotorControl2(robotId,3,controlMode=p.POSITION_CONTROL,targetPosition=val)
    val = val +0.001

cubePos, cubeOrn = p.getBasePositionAndOrientation(robotId)
print(cubePos,cubeOrn)
p.disconnect()
```

Warm Regards,
Rohit Kumar J

**Comment by rohit-kumar-j** [comment]
Please ping me for any further clarifications. :smile:

**Comment by yanshil** [comment]
Thank you for pointing this out. 

I am currently busying with some other staff, and I had some problem running PyBullet with my WSL, so fix might not be able to be applied very soon.

Welcome to send a pull request if you successfully fix this.

---



I feel like  there is an update of xyz calculation in Fusion 360 and caused this problem. Here is some quick fix I would recommend you to try. 

I checked syuntoku14's repo, and he had an update on core/Joint.py in 2020/01/19, [core/Joint.py](https://github.com/syuntoku14/fusion2urdf/commit/8786e6318cdcaaf32070148451a27ab6e4f6697d)

It might work (I had to set up my PyBullet again to test it. But.... I might not have time in these few weeks..)
